tcks/pom.xml

```diff
@@ -83,7 +83,7 @@
                 <groupId>org.webjars</groupId>
                 <artifactId>jquery</artifactId>
                 <!-- /!\ this version should not be upgraded, TestNG requires this particular version -->
-                <version>3.5.1</version>
+                <version>3.6.4</version>
```

Comment:
Don't merge this as is because we don't want to change this value.

Reply:
Reverted this change and force-pushed